docs/zh/UserGuide/IoTDB-SQL-Language/DML-Data-Manipulation-Language.md

```diff
@@ -432,6 +432,43 @@ Total line number = 5
 It costs 0.014s
 ```
 
+#### 常序列生成函数
+
+常序列生成函数用于生成所有数据点的值都相同的时间序列。
+
+常序列生成函数接受一个或者多个时间序列输入，其输出的数据点的时间戳集合是这些输入序列时间戳集合的并集。
+
+目前 IoTDB 支持如下常序列生成函数：
+
+| 函数名 | 输入序列类型                                    | 必要的属性参数                                               | 输出序列类型               | 功能描述                                                     |
```

Comment:
As we only care about the union of the timestamps of each input timeseries, the data type doesn't matter. So the docs can remove the 'Allowed Input Series Data Types' column. What do you think?

Reply:
Good idea!